src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -1052,3 +1057,29 @@ def create(self, request, *args, **kwargs):
 
         output_serializer = self.serializer_class(instances, many=True)
         return Response(output_serializer.data, status=status.HTTP_201_CREATED)
+
+
+class DocumentRegistrerenViewSet(viewsets.ViewSet):
+    queryset = EnkelvoudigInformatieObject.objects.all()
+    serializer_class = DocumentRegistrerenSerializer
+    permission_classes = (DocumentReserverenAuthRequired,)
+
+    # TODO  only used for api spec and `|` should be `&`
+    required_scopes = {
+        "create": SCOPE_DOCUMENTEN_AANMAKEN
+        | (SCOPE_ZAKEN_CREATE | SCOPE_ZAKEN_GEFORCEERD_BIJWERKEN),
+    }
+
+    viewset_classes = {
+        "enkelvoudiginformatieobject": "openzaak.components.documenten.api.viewsets.EnkelvoudigInformatieObjectViewSet",
+        "zaakinformatieobject": "openzaak.components.zaken.api.viewsets.ZaakInformatieObjectViewSet",
+    }
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.serializer_class(
+            data=request.data, context={"request": request}
+        )
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+
```

Comment:
Could you also add a `logger.info("document_geregistreerd", enkelvoudiginformatieobject_url=..., zaak_url=...)` statement here and mention that `document_geregistreerd` log event in `logging.rst`?